CMakeLists.txt

```diff
@@ -36,7 +36,11 @@ OPTION(BUILD_EXAMPLES "Build examples for carma" OFF)
 # ##############################################################################
 #                                   TARGET                                     #
 # ##############################################################################
-SET(PROJECT_RELEASE_FLAGS "-march=native" "-mtune=native")
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" 
+        OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang" 
+        OR CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
+    SET(PROJECT_RELEASE_FLAGS "-march=native" "-mtune=native")
```

Comment:
I only use it for GCC, Clang, and AppleClang.
Moreover, these options are good to get the best code optimization but it reduces the portability of the generated binaries.